src/client/storage_estimator/common/vos_size.py

```diff
@@ -237,19 +249,21 @@ def init_value(self, cont, akey, value_spec):
             nvme = False
         csum_size = cont["csum_size"]
         if csum_size != 0:
-            if akey["key"] == "single_value" or size < cont["csum_gran"]:
+            if akey["key"] == "single_value":
                 size += csum_size
             else:
-                size += (size // cont["csum_gran"]) * csum_size
-                if value_spec.get("aligned", "Yes") == "No":
-                    size += (csum_size * 2)
-        akey["count"] += value_spec.get("count", 1) # Number of values
+                size += int(math.ceil(size / cont["csum_gran"]) * csum_size)
+
+        akey["count"] += value_spec.get("count", 1)  # Number of values
         if value_spec.get("overhead", "user") == "user":
-            akey["value_size"] += size * value_spec.get("count", 1) # total size
+            akey["value_size"] += size * \
+                value_spec.get("count", 1)  # total size
         else:
-            akey["meta_size"] += size * value_spec.get("count", 1) # total size
+            akey["meta_size"] += size * \
+                value_spec.get("count", 1)  # total size
         if nvme:
-            akey["nvme_size"] += size * value_spec.get("count", 1) # total size
+            akey["nvme_size"] += size * \
```

Comment:
the checksum should always be added to meta_size only.  It is stored in SCM

Reply:
Done